README.md

```diff
@@ -22,9 +22,28 @@ Then navigate to the `topicalguide` directory and optionally switch over to the
 
 ### 2. Install Dependencies
 
+Superuser permissions are necessary to install the Topical Guide as shown in the current instructions.
```

Comment:
Maybe here we should just say something about how we are using virtualenv to avoid the need for superuser permissions? It sounds like we are telling the user that you actually need superuser instructions to do what is talked about below.

Also, do we still use virtualenv? I think Python3 uses pyvenv, or something like that, instead.